src/integ_test_resources/ios/amplify-ios/integration/cdk/cdk_integration_tests_ios/integration_test_resources/s3_construct.py

```diff
@@ -0,0 +1,31 @@
+from aws_cdk import aws_s3, core
+
+class S3Construct(core.Construct):
+    def __init__(
+        self, 
+        scope: core.Construct, 
+        construct_id: str, 
+        bucketPrefix: str,
+        **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        logging_bucket_name = bucketPrefix + "amplify-ios-integ-access-logging"
+        access_log_bucket = aws_s3.Bucket(
+            self,
+            "amplify-ios-integ-access-logging",
+            bucket_name=logging_bucket_name,
+            block_public_access=aws_s3.BlockPublicAccess.BLOCK_ALL,
+            removal_policy=core.RemovalPolicy.DESTROY,
+        )
+
+        self.bucket_name = bucketPrefix + "amplify-ios-integ"
```

Comment:
Does this need to be an instance variable since you're only referencing it from the constructor?

Reply:
Removed